code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -94,7 +95,7 @@
 	blood_color = "#b3cbc3"
 	base_color = "#066000"
 
-	//reagent_tag = IS_UNATHI //VOREStation Edit
```

Comment:
Unless you understand what this does, change this back, because I'm pretty sure all this does is make Unathi sick for eating non-meats, which is aggravating as hell.

Reply:
The effects are:
- Unathi get less hunger satisfaction from Nutriment
- Unathi get more hunger satisfaction from Protein
- Sweets (Sugar, Honey, Juice, Milkshake) make Unathi sleepy
- More resistant against alcohol